### PR TITLE
Removing inline CSS from handlebars and adding it to SASS

### LINKED ISSUE #990: SASS optimizations
- [ ] Remove redundancy by using SASS mixins
- [ ] Adding repeated values like padding, margin, colors etc to config.scss
- [x] Removing inline CSS from handlebars and adding it to SASS.

### PR DESCRIPTION
**Pull request by utkarshnath**
Fixed #990 (c)
@aayusharora please review

### CODE REVIEW

File: src/backend/templates/tracks.hbs [hunk 1]
```diff
@@ -109,47 +110,47 @@
...
-  		             	            </ul>
-  		             	            <ul style="padding:0;font-size:15px;">{{type}}</ul>
-                                </div>
-		                          </div>
-                            </div>
+			                      <div id="desc2-{{session_id}}" class="collapse in" style="background-color: {{../color}}; margin-top:0px;">
+                                      <div class="row">
+                                          <div class="col-md-6 col-xs-12 speakers-list">
+				                      {{#speakers_list}}
+				                      <p style="margin-right:10px; float:left" class="session-speakers">
```
Comment (championswimmer) [inline src/backend/templates/tracks.hbs]: would have been good if you didn't add further whitespace pollution
(the file is already pretty whitespace polluted)

**Comment by championswimmer** [comment]
LGTM (other than the whitespace issue commented above)

**Comment by SparshBansal** [comment]
@championswimmer @aayusharora We have removed the whitespace problems and formatted the code. Please review

**Comment by aayusharora** [comment]
@utkarshnath Can you provide a hosted link, to ensure that the style does not break.

**Comment by utkarshnath** [comment]
@aayusharora plz check 
https://mysterious-cove-37437.herokuapp.com/

**Pull request merged**